### PR TITLE
mujmap: init at 0.1.1

### PR DESCRIPTION
**Pull request by elizagamedev**
-------

###### Description of changes

[mujmap](https://github.com/elizagamedev/mujmap) is an email synchronization tool that I've recently published. As a NixOS user, I would like to take ownership of this package in nixpkgs.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/networking/mujmap/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  cargoSha256 = "sha256-nrLorxdJdQvCG8WhfsvHtWizrkzD38XqgeNyksKbvN4=";
+
+  buildInputs = [
+    notmuch
+  ] ++ lib.optional stdenv.isDarwin Security;
+
+  meta = with lib; {
+    description = "Bridge for synchronizing email and tags between JMAP and notmuch";
+    homepage = "https://github.com/elizagamedev/mujmap/";
+    license = licenses.gpl3;
```
Comment (fabaff) [inline pkgs/applications/networking/mujmap/default.nix:29]: According to be [file header](https://github.com/elizagamedev/mujmap/blob/main/src/args.rs#L7), the license is GPLv3+.

```suggestion
    license = licenses.gpl3Plus;
```

**Comment by elizagamedev** [comment]
Should I do anything further for this PR? Add other reviewers, etc? It seems like ofborg has been pending for a while now, I'm not sure if that's common.

**Comment by teto** [comment]
rebase to trigger ofborg again. The ability to synchronize my notmuch tags outside of gmail (fastmail with jmap) is the thing I miss the most on my various devices. I will try to set mujmap up and if I can make it work merge. I can deal with the home-manager module as well.

**Comment by elizagamedev** [comment]
> rebase to trigger ofborg again. The ability to synchronize my notmuch tags outside of gmail (fastmail with jmap) is the thing I miss the most on my various devices. I will try to set mujmap up and if I can make it work merge. I can deal with the home-manager module as well.

I have rebased. I'm glad to hear that there are NixOS contributors who would find value from mujmap :)

**Pull request merged**